tests/containers/rootless_podman.pm

```diff
@@ -56,6 +56,7 @@ sub run {
     # already exists owned by root
     assert_script_run 'rm -rf /tmp/script*';
     ensure_serialdev_permissions;
+    assert_script_run 'loginctl enable-linger 1000';
```

Comment:
Better safe than sorry :wink: 
```suggestion
    assert_script_run 'loginctl enable-linger $(id -u bernhard)';
```

Reply:
even better if _bernhard_ is replaced by `$user`